packages/cli/generators/copyright/header.js

```diff
@@ -0,0 +1,287 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+const _ = require('lodash');
+const git = require('./git');
+const path = require('path');
+const fs = require('fs-extra');
+const Project = require('@lerna/project');
+
+const {promisify} = require('util');
+const glob = promisify(require('glob'));
+
+const debug = require('debug')('loopback:cli:copyright');
+
+// Components of the copyright header.
+const COPYRIGHT = [
+  'Copyright <%= owner %> <%= years %>. All Rights Reserved.',
+  'Node module: <%= name %>',
+];
+const LICENSE = [
+  'This file is licensed under the <%= license %>.',
+  'License text available <%= ref %>',
+];
```

Comment:
Have you considered using ES6 template literals instead of lodash templates?

```ts
function license({license, ref}) {
  return [
    `This file is licensed under the ${license}.`,
    `License text available ${ref}`,
  ];
}
```

Reply:
It's a bit involved. We can defer it for future improvement.